deployment/build_and_release/modules/release/variables.tf

```diff
@@ -110,6 +110,12 @@ variable "os_public_key2" {
   EOT
 }
 
+variable "console" {
+  description = "If set to `on`, then build with the `console` build arg"
```

Comment:
Could say "If set to `on`, then the bootloader firmware will emit debug logging"?

Reply:
Thanks!